cmake-tests/CMakeLists.txt

```diff
@@ -1,8 +1,9 @@
+set(CMAKE_CXX_COMPILER "${PROJECT_BINARY_DIR}/compiler/bin/clang++")
 
-if(COMMAND CMAKE_FORCE_CXX_COMPILER)
-CMAKE_FORCE_CXX_COMPILER("${PROJECT_BINARY_DIR}/compiler/bin/clang++" MCWAMPCC)
-endif()
 add_executable(cmake-test cmake-test.cpp)
+
+add_dependencies(cmake-test world)
```

Comment:
This breaks the test when ran as standalone, which is useful for testing the `find_package(hcc)` logic. Either this line should be moved into the `if(TARGET hccrt)` conditional block or hccrt should be update to depend on the world target, so this line can be removed.